template/amber/repos.amber

```diff
@@ -22,8 +22,7 @@ block content
                             div.card-header
                                 img.avatar[src=$repo.Avatar]
                             div.card-block
-                                h3.login #{$repo.Name}
-                                div.full_name.hidden #{$repo.FullName}
+                                h3.full_name #{$repo.FullName}
```

Comment:
No, changing it from `.login` to `.full_name` does not alter the style. None of the styles for `.login` apply here (I'm not sure why it has that attribute set at all) and `.full_name` does not have any styling at all.

However the search box depends on  `#{$repo.FullName}` having a class of `full_name`.